backend/private-graph/graph/schema.graphqls

```diff
@@ -1176,6 +1176,7 @@ type WorkspaceInviteLink {
 	invitee_role: String!
 	expiration_date: Timestamp!
 	secret: String!
+	created_at: Timestamp!
```

Comment:
Do we need this, or is the preferred way to access through `Model`?

Reply:
should be right - this will expose the attribute from the `Model` base